### PR TITLE
fix job `CRD metadata.annotations: Too long` error

### PR DESCRIPTION
**Pull request by Monokaix**
Fix https://github.com/volcano-sh/volcano/issues/3107

### CODE REVIEW

**Review by hwdef — COMMENTED** [review]
I don't think this repair is good, because when the yaml is regenerated, it will be overwritten.

**Comment by Monokaix** [comment]
> I don't think this repair is good, because when the yaml is regenerated, it will be overwritten.

yeah, it's a question, `kubectl apply --server-side` is another way.

**Comment by Monokaix** [comment]
> I don't think this repair is good, because when the yaml is regenerated, it will be overwritten.

Limit the volcano job crd yaml's description when using CONTROLLER_GEN, please check it once more.

**Review by hwdef — COMMENTED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,7 @@ RELEASE_DIR=_output/release
 REPO_PATH=volcano.sh/volcano
 IMAGE_PREFIX=volcanosh
 CRD_OPTIONS ?= "crd:crdVersions=v1,generateEmbeddedObjectMeta=true"
+CRD_OPTIONS_EXCLUDE_DESCRIPTION=${CRD_OPTIONS}",maxDescLen=0"
```
Comment (william-wang) [inline Makefile:20]: Is there any effect after removing the description?

Comment (Monokaix) [reply Makefile:20]: description comes from the comment of CRD'field, and removing it has no effect to job's CRD definition, the only effect is that user use `kubectl get crd xxx -oyaml` cannot see the descrption of every field and then can check code or doc to get detail description of every field.

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**